net/wireshark2/Portfile

```diff
@@ -4,14 +4,20 @@ PortSystem          1.0
 PortGroup           cmake 1.0
 
 name                wireshark2
-version             2.6.19
-revision            2
+version             2.6.20
+revision            0
 categories          net
 license             {GPL-2 GPL-3}
 maintainers         {darkart.com:opendarwin.org @ghosthound}
 description         Graphical network analyzer and capture tool
 homepage            https://www.wireshark.org/
-platforms           darwin
+# macOS 10.12+ are supported with Wireshark 3.x/4.x; 10.11 and below are limited to 2.x.
```

Comment:
Does "macOS 10.12+" mean "any macOS version from 10.12 to latest" or "any macOS 10.12.x version"?  I think the latter, and if so the comment should be adjusted to be clearer that we're only talking about macOS 10.12.x here.

Reply:
@ghosthound Please take a look, I have modified the comment.